functests/utils/consts.go

```diff
@@ -81,3 +81,8 @@ const (
 	// ContainerMachineConfigDaemon contains the name of the machine-config-daemon container
 	ContainerMachineConfigDaemon = "machine-config-daemon"
 )
+
+const (
+	// RTKernel contains the line that the RT kernel should contain under the uname -a
+	RTKernel = "PREEMPT_RT"
```

Comment:
we should investigate why this value changed in the first place. Different patchset? This can be done once CI is unblocked, but we should not forget about this.